bootstrap/secret/secret.go

```diff
@@ -171,7 +171,7 @@ func getSecretConfig(secretStoreInfo *config.SecretStoreInfo,
 		Type:                 secretStoreInfo.Type, // Type of SecretStore implementation, i.e. Vault
 		Host:                 secretStoreInfo.Host,
 		Port:                 secretStoreInfo.Port,
-		SecretName:           addEdgeXSecretNamePrefix(secretStoreInfo.SecretName),
+		StoreBasePath:        addEdgeXSecretNamePrefix(secretStoreInfo.SecretName),
```

Comment:
My comments on SecretStoreInfo still need to be addressed

```suggestion
		StoreBasePath:        addEdgeXSecretNamePrefix(secretStoreInfo.StoreName),
```